CONTRIBUTING.md

```diff
@@ -26,31 +26,29 @@ Use IntelliJ IDEA Community Edition/Ultimate.
 
 3. (optional) If working on IntelliJ Ultimate, enable JavaScript and TypeScript plugin (for UI tests).
 
-4. (optional) Install the following non-bundled plugins from Marketplace:
+4. Install the following non-bundled plugins from Marketplace:
+    * [Kotlin plugin](https://plugins.jetbrains.com/plugin/6954-kotlin) will be useful for editing certain parts of UI, esp. dialogs.
+    * [Scala plugin](https://plugins.jetbrains.com/plugin/1347-scala) might be useful for editing UI tests.
+
+5. (optional) Install further non-bundled plugins from Marketplace:
    * [Grammar-Kit IntelliJ plugin](https://plugins.jetbrains.com/plugin/6606-grammar-kit) can be used instead of Gradle plugin
      to manually generate grammar and lexer code from `.bnf` and `.flex` files.
    * [HOCON plugin](https://plugins.jetbrains.com/plugin/10481-hocon) for `.conf` file support in UI tests
-   * [Kotlin plugin](https://plugins.jetbrains.com/plugin/6954-kotlin) will be useful for editing certain parts of UI, esp. dialogs.
    * [PsiViewer IntelliJ plugin](https://plugins.jetbrains.com/plugin/227-psiviewer) can be helpful to see parsing result on the `machete` file
      when running IntelliJ instance with the Git Machete plugin loaded.
-   * [Scala plugin](https://plugins.jetbrains.com/plugin/1347-scala) might be useful for editing UI tests.
 
-5. Enable annotation processing (for Lombok):
+6. Enable annotation processing (for Lombok):
    `File > Settings > Build, Execution, Deployment > Compiler > Annotation Processors > Enable Annotation Processing`.
    Select `Obtain annotation processors from classpath` radio box.
 
-6. Set Project SDK to JDK 17: `Project Structure > Project`
-
-7. Consider increasing maximum heap size for the IDE (the default value is 2048 MB) under `Help > Change Memory Settings`.
+7. (optional) Increase maximum heap size for the IDE (the default value is 2048 MB) under `Help > Change Memory Settings`.
 
-8. For running `./gradlew` from the command line, make sure that `java` and `javac` are in `PATH` and point to Java 17.
-
-9. Consider [enabling internal mode](https://plugins.jetbrains.com/docs/intellij/enabling-internal.html).
+8. (optional) [Enable internal mode](https://plugins.jetbrains.com/docs/intellij/enabling-internal.html).
    It can be significantly useful while working with UI components (or tests).
    To investigate the UI you may want to use `Tools > Internal Actions > UI > UI Inspector`.
 
-10. Consider setting `Class count before import with '*'` and `Names count to use static import with '*'` to a very high number (e.g. 500)
-to avoid problems with CheckStyle when editing code. To do that go to `Preferences > Code Style > Java/Kotlin/Scala`.
+9. (optional) Set `Class count before import with '*'` and `Names count to use static import with '*'` to a very high number (e.g. 500)
+   to avoid problems with CheckStyle when editing code. To do that go to `File > Settings > Code Style > Java/Kotlin/Scala`.
```

Comment:
Same here - but `Preferences > Code Style > Java/Kotlin/Scala` is **not valid** for macOs. The valid path is `Preferences > Editor > Code Style > Java/Kotlin/Scala`. Looks like it we should fix this in a separate issue.

Reply:
+1